reference-implementation/to-upstream-wpts/writable-streams/bad-underlying-sinks.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+  self.importScripts('../resources/test-utils.js');
+  self.importScripts('../resources/recording-streams.js');
+}
+
+const error1 = new Error('error1');
+error1.name = 'error1';
+
+promise_test(t => {
+
```

Comment:
Personally I write the tests with blank lines after the opening brace and before the last line/description. Otherwise it's too hard to distinguish the test description from the test code. *shrug*